recipe/meta.yaml

```diff
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
     - versioneer
-    - tomli # [py<311]
```

Comment:
Should `toml` be added here because of [this](https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml#L15)?

Reply:
I think just versioneer is ok